### ISSUE TITLE
1.2.0_k8s_istio | error applying application istio-stack

### ISSUE THREAD

**Issue by romanzdk** [kubeflow/kubeflow#5677]
/kind bug

I am trying to install kubeflow 1.2.0 on Azure AKS.
Used command:
```kfctl apply -V -f https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_k8s_istio.v1.2.0.yaml```

Problem:
```WARN[0062] Encountered error applying application istio-stack:  (kubeflow.error): Code 500 with message: Apply.Run : error when creating "/tmp/kout866647084": Internal error occurred: failed calling webhook "validation.istio.io": Post https://istiod.istio-system.svc:443/validate?timeout=30s: service "istiod" not found  filename="kustomize/kustomize.go:284" WARN[0062] Will retry in 18 seconds. filename="kustomize/kustomize.go:285"```

EDIT: I also tried:

- 1.1, 1.0.2 and 1.0.1 versions but still unsuccessfully.
- installing on kubernetes 1.16.1 and still the same.
- k8s_istio_dex 1.2.0 and still the same,
- installing it on minikube but I got again similar error:

```WARN[0043] Encountered error applying application cert-manager:  (kubeflow.error): Code 500 with message: Apply.Run : error when creating "/tmp/kout465406925": Internal error occurred: failed calling webhook "webhook.cert manager.io": the server is currently unable to handle the request filename="kustomize/kustomize.go:284" WARN[0043] Will retry in 10 seconds. filename="kustomize/kustomize.go:285"```

**What did you expect to happen:**
Successfully installed kubeflow.

**Environment:**

- Kubeflow version: 1.2.0
- kfctl version: kfctl v1.2.0-0-gbc038f9
- Kubernetes platform: Azure AKS
- Kubernetes version: Client 1.20.4 | Server 1.18.14
- OS: macOS Catalina 10.15.7

**Comment by jbottum** [comment]
@romanzdk Hey Roman, did you get this resolved?

**Comment by romanzdk** [comment]
No, I have not

**Comment by jbottum** [comment]
@aronchick hey david, is there someone that might help on this question ?

**Comment by glong1997** [comment]
I met the same problem.

**Comment by mavencode01** [comment]
Same issue installing on Openshift: 

![istio_gallery](https://user-images.githubusercontent.com/542912/118892529-05348080-b8cf-11eb-802f-40e15060178a.png)

**Comment by muhammadtehseenSystemsltd** [comment]
Same problem here

**Comment by wangjun1974** [comment]
same problem on Openshift 4.5.18 with kfctl_openshift.v1.2.0.yaml

**Comment by jbottum** [comment]
@nakfour please see questions on OpenShift and Kubeflow in this issue

**Comment by namlh16** [comment]
> did you get this resolved? and please how?

@mavencode01

**Comment by lions1988** [comment]
Hello all,
Did you anyone got this resolved? deployment failed on kubeflow 1.3 on top of Openshift 4.10
I've tried to deploy using Opendatahub, as well using kfctl - both failed with the same error  
```
Encountered error applying application istio-stack: (kubeflow.error): Code 500 with message: Apply.Run
```

As well as several pods under istio-system namespace crash looping

Did anyone were able to deploy Kubeflow on top of latest Openshift?

**Comment by juliusvonkohout** [comment]
/close

There has been no activity for a long time. Please reopen if necessary.
Please also consult the Kubeflow slack channel for support questions.
